client/settings/settings-manager/style.scss

```diff
@@ -0,0 +1,49 @@
+body.woocommerce_page_wc-settings-checkout,
+body.woocommerce_page_wc-settings-checkout .settings-content,
+body.woocommerce_page_wc-settings-checkout #wpbody-content,
+body.woocommerce_page_wc-settings-checkout #wpcontent {
+	background: #fff !important;
```

Comment:
Is the `!important` really necessary? The selectors should be specific enough to override the default color.

Reply:
yes, since the styling from core has priority in this case. I added a comment to remove it when all Settings section will use the new design.

Reply:
I believe we can set a more specific selectors where the !important is not needed, like:
```
body.woocommerce_page_wc-settings.woocommerce_page_wc-settings-checkout,
body.woocommerce_page_wc-settings.woocommerce_page_wc-settings-checkout .settings-content,
body.woocommerce_page_wc-settings.woocommerce_page_wc-settings-checkout #wpbody-content,
body.woocommerce_page_wc-settings.woocommerce_page_wc-settings-checkout #wpcontent {
```

Reply:
pushed in [76e18d0](https://github.com/Automattic/woocommerce-payments/pull/10578/commits/76e18d030465eab6a4c22ce9129467215fdca4fc) . thanks .